aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/question_list.html

```diff
@@ -9,7 +9,7 @@
 {% endblock %}
 
 {% block faq_footer %}
-    {% url 'aldryn_faq:faq-category-list' as category_url %}
-    {% include "aldryn_faq/includes/pager.html" with title=_("Back to Overview") slug=category_url %}
+    {% url 'aldryn_faq:faq-category-list' as category_list_url %}
+    {% include "aldryn_faq/includes/pager.html" with title=_("Back to Overview") slug=category_list_url %}
```

Comment:
agree ```slug``` is not really the best, this comes from the initial bootstrap migration.. I can rename but then we need to do something like {% firstof %} on the pager template to make sure we're still compatible with the old variable